doc/reference/configuration/index.rst

```diff
@@ -99,40 +104,76 @@ If ``username:password`` is omitted, then 'guest' is assumed. Some examples:
     | username:password@host:port | notguest:sesame@mail.ru:3301 |
     +-----------------------------+------------------------------+
 
+In code, the URI value can be passed as a number (if only a port is specified) or a string:
+
+..  code-block:: lua
+
+    box.cfg { listen = 3301 }
+
+    box.cfg { listen = "127.0.0.1:3301" }
+
 In certain circumstances, a Unix domain socket may be used
 where a URI is expected, for example, "unix/:/tmp/unix_domain_socket.sock" or
 simply "/tmp/unix_domain_socket.sock".
 
+A method for parsing URIs is illustrated in :ref:`Module uri <uri-parse>`.
+
+.. _index-uri-several:
+
+Specifying several URIs
+~~~~~~~~~~~~~~~~~~~~~~~
+
 Starting from version 2.10.0, a user can open several listening iproto sockets on a Tarantool instance
-and, consequently, can specify several URIs in the configuration parameters such as :ref:`box.cfg.listen <cfg_basic-listen>` and :ref:`box.cfg.replication <cfg_replication-replication>`.
+and, consequently, can specify several URIs in the configuration parameters
+such as :ref:`box.cfg.listen <cfg_basic-listen>` and :ref:`box.cfg.replication <cfg_replication-replication>`.
 
-In addition to specifying URI as a number or a string, a number of other ways to pass URI values is available starting from Tarantool 2.10.0:
+URI values can be set in a number of ways:
 
-*   As a string with one or several URIs separated by commas (provides backward compatibility)
+*   As a string with URI values separated by commas
 
-    code-block:: lua
+    ..  code-block:: lua
 
-    box.cfg { listen = "127.0.0.1:3301, /unix.sock, 3302" }
+        box.cfg { listen = "127.0.0.1:3301, /unix.sock, 3302" }
 
 *   As an array that contains URIs in a string format
 
-    code-block:: lua
+    ..  code-block:: lua
 
-    box.cfg { listen = {"127.0.0.1:3301", "/unix.sock", "3302"} }
+        box.cfg { listen = {"127.0.0.1:3301", "/unix.sock", "3302"} }
 
 *   As an array of tables with the ``uri`` field
 
-    code-block:: lua
+    ..  code-block:: lua
 
-    box.cfg { listen = {
-            {uri = "127.0.0.1:3301"},
-            {uri = "/unix.sock"},
-            {uri = 3302}
+        box.cfg { listen = {
+                {uri = "127.0.0.1:3301"},
+                {uri = "/unix.sock"},
+                {uri = 3302}
+            }
         }
+
+Also, you can specify additional URI parameters. The recommended syntax:
```

Comment:
Here i would indicate all the methods as in the previous comment with the recommended ones

Reply:
Examples are added.